application/src/test/java/org/thingsboard/server/transport/lwm2m/AbstractLwM2MIntegrationTest.java

```diff
@@ -170,12 +173,19 @@
     protected final Certificate[] trustedCertificates = new Certificate[1];
 
     protected static final int SECURE_PORT = 5686;
+    protected static final int SECURE_PORT_BS = 5688;
+    protected static final String HOST = "localhost";
+    protected static final String HOST_BS = "localhost";
     protected static final NetworkConfig SECURE_COAP_CONFIG = new NetworkConfig().setString("COAP_SECURE_PORT", Integer.toString(SECURE_PORT));
-    protected static final String ENDPOINT = "deviceAEndpoint";
+    protected static final String ENDPOINT = "deviceEndpoint";
```

Comment:
Renaming the endpoint may cause issue with predefined X509 credentials we use in test.